filebeat/input/journald/input.go

```diff
@@ -112,6 +115,21 @@ func Configure(cfg *conf.C, _ *logp.Logger) ([]cursor.Source, cursor.Input, erro
 		sources[i] = pathSource(p)
 	}
 
+	if config.Chroot != "" {
+		chrootStat, err := os.Stat(config.Chroot)
+		if err != nil {
+			return nil, nil, fmt.Errorf("cannot stat chroot:  %w", err)
+		}
+		if !chrootStat.IsDir() {
+			return nil, nil, fmt.Errorf("provided chroot (%s) is not a directory", config.Chroot)
+		}
+
+		fullPath := filepath.Join(config.Chroot, config.JournalctlPath)
```

Comment:
We may want to verify the ownership of the file, maybe that it is only writable by root so it can't be replaced by something from an arbitrary user, especially if we are going to execute it as root.

Speaking of executing it as a root, if we can execute this as a lower privileged user that would help the security boundary here too.